editwatcher.py

```diff
@@ -0,0 +1,160 @@
+# coding=utf-8
+import json
+import os.path
+import requests
+import time
+import threading
+# noinspection PyPackageRequirements
+import websocket
+# noinspection PyPackageRequirements
+from bs4 import BeautifulSoup
+from urllib.parse import urlparse
```

Comment:
import sort order: urllib.parse should be up among the standard library imports

Reply:
Basically the same reply here. I'm happy to reorder these `import` statements and apply the style changes you mention, but the current ordering and style is something which is in the existing deletionwatcher.py.

The thing I didn't do, and definitely should have done, is go through each import and remove the ones which aren't actually being used, at least both here in editwatcher.py and remove those which are no longer needed in deletionwatcher.py.

I'll go ahead and make the changes you're asking for along with what I mentioned above.